src/storage.js

```diff
@@ -134,9 +136,62 @@ class HybridStore extends MemoryStore {
     await super.cleanStorage(cleanHistory, cleanAddresses, cleanTokens);
     if (cleanTokens) {
       STORE.removeItem(REGISTERED_TOKENS_KEY);
+      STORE.removeItem(REGISTERED_NANOCONTRACTS_KEY);
     }
   }
 
+  /**
+   * Iterate on registered nano contracts.
+   *
+   * @async
+   * @returns {AsyncGenerator<INcData>}
+   */
+  async* registeredNanoContractsIter() {
+    const registeredNanoContracts = STORE.getItem(REGISTERED_NANOCONTRACTS_KEY) || {};
+    for (const ncData of Object.values(registeredNanoContracts)) {
+      yield ncData;
+    }
+  }
+
+  /**
+   * Register a nano contract.
+   *
+   * @param ncId {string} Nano contract id to register
+   * @param ncValue {INcData} Nano contract data to register
+   * @async
+   * @returns {Promise<void>}
+   */
+  async registerNanoContract(ncId, ncValue) {
+    await super.registerNanoContract(ncValue);
```

Comment:
question(blocking): Are we missing the `ncId` parameter here?

```suggestion
    await super.registerNanoContract(ncId, ncValue);
```

Reply:
Definitely. This was a bug, thanks.